pages/common/wp.md

```diff
@@ -0,0 +1,24 @@
+# wp
+
+> The official command-line interface to manage WordPress instances.
+> More information: <https://wp-cli.org/>.
+
+- Describe the operating system, shell, PHP, and WP-CLI installation:
```

Comment:
Add an appositive to clarify? (assuming `WP-CLI` _is_ `ws`)
```suggestion
- Describe the operating system, shell, PHP, and WP-CLI (`ws`) installation:
```

Reply:
Have I done something wrong somewhere?

Where is `ws` coming from? ^-^' @CleanMachine1 also mentioned `ws`, but I assumed it was just a typo of `wp`.

Reply:
it was

Reply:
Oops, I think I was getting it from @CleanMachine1's comment 😄

Suggestion updated.